### ISSUE TITLE
False positiv in no-unused-expressions

### ISSUE THREAD

**Issue by ptitgraig** [eslint/eslint#1936]
Hello.
I've run eslint on some code, and found that the rule `no-unused-expressions` raises an error with the following code while,i think, it shouldn't. Do you confirm it's a false positiv ?

```javascript
  var found, index;
  found = false,
  index = 0;
```

**Comment by gyandeeps** [comment]
The idea is it will flag a variable to which you just assign values but you never end up using them anywhere.
So in above code it will flag them as unused.

**Comment by nzakas** [comment]
Can you paste in the errors you're seeing?

**Comment by ptitgraig** [comment]
The error i'm seing :
```shell
699:16   warning  Expected an assignment or function call and instead saw an expression  no-unused-expressions
```

The related line of code (709) :

```javascript
isShiftTab: function(arr, ele) {
  var found, index;
  found = false,
  index = 0;
  while (!found && index < arr.length) {
    if (arr[index] === ele) {
      found = true;
    } else {
      index++;
    }
  }
  return found;
},
```

**Comment by michaelficarra** [comment]
The listed line number is 699, not 709. And I don't see any line numbers in your snippet.

**Comment by ptitgraig** [comment]
Indeed, sorry for the bad copy paste. I made the correction in my previous post.
I'm not sure how to add line numbers though with markdown. Line 699 is the line with `found = false,`

**Comment by michaelficarra** [comment]
I see. You have a comma instead of a semicolon, so it is a sequence in statement position rather than two assignments in statement position.

**Comment by nzakas** [comment]
Duplicate of #1937
